packages/user-mgnt/src/config/routes.ts

```diff
@@ -18,9 +18,6 @@ import changeAvatarHandler, {
   changeAvatarRequestSchema
```

Comment:
In the user-mgnt service, the most significant change is that the concept of `scope` has now been removed completely thus making assigning scopes to users and roles the responsibility of the auth service

Reply:
In this file I've just updated the Hapi syntax a bit and removed the, now reduntant, `getSystemRoles` method